### PR TITLE
Retrieve oauth client ID and secret from env vars

### PR DESCRIPTION
**Pull request by danparsons**
Updated README.md to reflect this too. This commit includes a fix for the previous PR that failed build tests.

### CODE REVIEW

**Comment by danparsons** [comment]
Whoops! I shall try again...

File: lib/cf/bridge/src/index.js [hunk 1]
```diff
@@ -293,7 +293,9 @@ const stopReporting = () => {
 };
 
 const scheduleUsageReporting = () => {
-  oauth.start('bridge', 'secret');
+  var clientId = process.env.CLIENT_ID ? process.env.CLIENT_ID : "bridge";
```
Comment (hsiliev) [inline lib/cf/bridge/src/index.js]: I personally would prefer to have no hardcoded user/pass. If you go this direction you will need to modify the cf-bridge integration test in tests/cf/bridge

Comment (danparsons) [inline lib/cf/bridge/src/index.js]: I would prefer that too, but it had hard coded values originally, I assumed they were there on purpose?

Comment (hsiliev) [inline lib/cf/bridge/src/index.js]: This was the easiest way to have the bridge running. We had the idea to make this configurable and use the oauth module in abacus (https://www.pivotaltracker.com/story/show/108061970).

File: lib/cf/bridge/src/index.js [hunk 2]
```diff
@@ -293,7 +293,9 @@ const stopReporting = () => {
...
+  var secret = process.env.SECRET ? process.env.SECRET : "secret";
```
Comment (hsiliev) [inline lib/cf/bridge/src/index.js]: You can use `const` instead of `var`

**Pull request closed without merging**